src/app/3d/qgs3dmaptoolidentify.cpp

```diff
@@ -78,7 +81,18 @@ void Qgs3DMapToolIdentify::mouseReleaseEvent( QMouseEvent *event )
     {
       const QgsRayCastingUtils::RayHit hit = it->second.first();
       const QgsVector3D mapCoords = Qgs3DUtils::worldToMapCoordinates( hit.pos, mCanvas->mapSettings()->origin() );
-      const QgsPoint pt( mapCoords.x(), mapCoords.y(), mapCoords.z() );
+      QgsVector3D mapCoordsCanvas2D;
+      try
+      {
+        mapCoordsCanvas2D = ct.transform( mapCoords );
+      }
+      catch ( QgsException &e )
+      {
+        Q_UNUSED( e )
```

Comment:
`e` is used in the next line.

Reply:
It's not always used! `QgsDebugError`'s parameter is ignored on non debug builds